src/main/java/org/gridsuite/filter/server/FilterService.java

```diff
@@ -135,6 +135,17 @@ public FilterService(FiltersToGroovyScript filtersToScript,
         }
     }
 
+    public void renameFilter(UUID id, String newName) {
```

Comment:
shouldn't it be @Transactional ?

Reply:
Why should it be ? it's only one request